File: GTTS/INL.TrainingService.Models/GetTrainingEventStudentRoster_Item.cs

namespace INL.TrainingService.Models
{
    public class GetTrainingEventStudentRoster_Item : IGetTrainingEventStudentRoster_Item
    {
        public long TrainingEventID { get; set; }
        public string TrainingEventName { get; set; }
        public long? TrainingEventGroupID { get; set; }
        public string TrainingEventGroupName { get; set; }
        public byte[] FileContent { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-training-event-student-roster_item.ts


import { IGetTrainingEventStudentRoster_Item } from './iget-training-event-student-roster_item';

export class GetTrainingEventStudentRoster_Item implements IGetTrainingEventStudentRoster_Item {
  
	public TrainingEventID: number = 0;
	public TrainingEventName: string = "";
	public TrainingEventGroupID?: number;
	public TrainingEventGroupName: string = "";
	public FileContent: number[] = null;
  
}






